### DISCUSSION TITLE
Restrict from-address per SMTP credential.

### DISCUSSION THREAD

**Discussion by ajithchanaka** [postalserver/postal#2130] [Help with using Postal]
Is there a method to restrict from-addresses per SMTP credentials? I want to block all from-addresses other than the allowed list per SMTP credentials. Someone help please?

**Comment by willpower232** [comment]
This is not currently a feature of Postal, it only controls which domains you can use to try and guarantee the message will be delivered.

**Reply by ajithchanaka** [reply]
Thank you for the reply.
